sdk/tests/get_tickmap.test.ts

```diff
@@ -0,0 +1,201 @@
+import { Keyring } from '@polkadot/api'
+import { assert } from 'chai'
+import { Invariant } from '../src/invariant'
+import { Network } from '../src/network'
+import { PSP22 } from '../src/psp22'
+import { assertThrowsAsync } from '../src/testUtils'
+import { initPolkadotApi, newFeeTier, newPoolKey } from '../src/utils'
+
+const api = await initPolkadotApi(Network.Local)
+
+const keyring = new Keyring({ type: 'sr25519' })
+const account = await keyring.addFromUri('//Alice')
+
+let invariant = await Invariant.deploy(api, Network.Local, account, { v: 10000000000n })
+let token0 = await PSP22.deploy(api, Network.Local, account, 1000000000n, 'Coin', 'COIN', 0n)
+let token1 = await PSP22.deploy(api, Network.Local, account, 1000000000n, 'Coin', 'COIN', 0n)
+
+describe('tickmap', async () => {
+  const feeTier = newFeeTier({ v: 10000000000n }, 1n)
+  const ticks = [-221818n, -221817n, -58n, 5n, 221817n, 221818n]
+  let poolKey = newPoolKey(
+    token0.contract.address.toString(),
+    token1.contract.address.toString(),
+    feeTier
+  )
+  beforeEach(async () => {
+    invariant = await Invariant.deploy(api, Network.Local, account, { v: 10000000000n })
+    token0 = await PSP22.deploy(api, Network.Local, account, 1000000000n, 'Coin', 'COIN', 0n)
+    token1 = await PSP22.deploy(api, Network.Local, account, 1000000000n, 'Coin', 'COIN', 0n)
+
+    poolKey = newPoolKey(
+      token0.contract.address.toString(),
+      token1.contract.address.toString(),
+      feeTier
+    )
+
+    await invariant.addFeeTier(account, feeTier)
+
+    await invariant.createPool(
+      account,
+      token0.contract.address.toString(),
+      token1.contract.address.toString(),
+      feeTier,
+      { v: 1000000000000000000000000n },
+      0n
+    )
+
+    await token0.approve(account, invariant.contract.address.toString(), 10000000000n)
+    await token1.approve(account, invariant.contract.address.toString(), 10000000000n)
+  })
+
+  it('get tickmap', async () => {
+    const pool = await invariant.getPool(
+      account,
+      token0.contract.address.toString(),
+      token1.contract.address.toString(),
+      feeTier
+    )
+    await invariant.createPosition(
+      account,
+      poolKey,
+      ticks[2],
+      ticks[3],
+      { v: 10n },
+      pool.sqrtPrice,
+      pool.sqrtPrice
+    )
+
+    const tickmap = await invariant.getTickmap(account, poolKey, pool.currentTickIndex)
+    assert.deepEqual(tickmap[3465], 9223372036854775809n)
```

Comment:
And all remaining chunks should be `0`, right? It's worth validating this. It could be helpful to avoid possible hidden errors